community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,58 @@
+# Best practices for managing onnx issues
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
+* ONNX Bug
+* ONNX Question
+* Propose New Feature, Operator or Enhancement
+
+
+## Label issues
+
+* Going forward all issues must be labeled. 
+* Multiple labels should be used to properly capture the issue. For example when a user logs a shape inference bug for a particular operator.  [bug][shape-inference] labels should be added.
+* Labels should use a consistent naming scheme
+* New label must be created for each release and it must be applied to all PRs and issues which are targeted for the release.
+* "backlog" label should be applied to all the issues which are cut from current release but are still on the radar to be addressed. This will prevent them from being closed as stale.
```

Comment:
If Milestone tags are used, it would simply be moved to the next Milestone. 
I think I would prefer having labels and milestones as different concepts; labels = categorization; milestone = time-bound release vehicle for fix

Reply:
I am adding this lable to enable excluding issues from being marked as stale by stale bot. 
I need to check whether Probot\stale can exclude issues with Milestone tag. If yes then I will remove this label.

Reply:
It can... I will remove this label